quarkus-jandex/README.md

```diff
@@ -0,0 +1,3 @@
+### Relevant Articles:
```

Comment:
We can omit this README file for now. The editor will create it after the article is published to ensure that it has the correct URL.